requirements-minimal.txt

```diff
@@ -20,6 +20,8 @@ humanfriendly
 humanize >= 0.5.1
 inotify >= 0.2.8
 ipaddress >= 1.0.22
+ipdb
+ipython<7
```

Comment:
why do we need them at the prod?

Reply:
we already have python repl at prod. this is just a better repl and a debugger, useful everywhere imo.